Cognigy/WebchatWidget
# Auto focus element in latest incoming message

This PR adds a feature flag 'enableAutoFocus', which when enabled will move the focus to the first interactive element within the latest incoming message. The focus will not be moved when the currently focused element is not within the chat history. This feature will be helpful to maintain the focus order for keyboard users. 

To test:
- Set enableAutoFocus to true in webchat settings
- Use a flow that includes a lot of interactive elements (List with links/ buttons, audio, video, gallery items with links/ buttons, quick replies with buttons)
- Interact with the bot. Only when an element within the chat log is currently focused, the focus should automatically move to the incoming message's first interactive element. That is, if the focus is currently on message input, chat toggle button, close chat button or somewhere outside the chat log/ chat window, the focus will not be moved automatically. 
- On clicking enter, the focused item should be activated.
- When enableAutoFocus is false, the focus should not be moved.
